cluster/cluster_test.go

```diff
@@ -0,0 +1,20 @@
+package cluster
+
+import (
+	"testing"
+
+	"github.com/astronomer/astro-cli/config"
+	testUtils "github.com/astronomer/astro-cli/pkg/testing"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestClusterDoesNotExists(t *testing.T) {
+	// fake filesystem
+	fs := afero.NewMemMapFs()
+	// we have only localhost domain in test config
+	configYaml := testUtils.NewTestConfig()
+	err := afero.WriteFile(fs, config.HomeConfigFile, []byte(configYaml), 0777)
+	config.InitConfig(fs)
+	assert.Equal(t, Exists("localhost123"), false, err)
```

Comment:
I'm confused by two things here:

1. where the `Exists()` fn is coming from - i don't see any wildcard imports or this defined anywhere?
2. what this is testing (my testify is rusty): Is that that localhost123 appears in the error? What's the `false` parameter for?

Reply:
1) in go you can use functions from same package in different files:
`cluster/cluster.go:57` https://github.com/astronomer/astro-cli/blob/66e4019ec93551b612c06acfb9682c5bf80ceed7/cluster/cluster.go#L57
2) what about?
```
	// Check that we don't have localhost123 in test config from testUtils.NewTestConfig()
	assert.Equal(t, Exists("localhost123"), false, err)
```